src/config/app/app.config.ts

```diff
@@ -4,4 +4,5 @@ export default registerAs("app", () => ({
     nodeEnv: process.env.NODE_ENV,
     port: process.env.PORT,
     frontendUrl: process.env.FRONTEND_URL,
+    latestReleaseLink: process.env.LATEST_RELEASE,
```

Comment:
I would update this variable link to LATEST_RELEASE_LINk so it's clear that we're putting a link here not the version number